### PR TITLE
feat: adding Popover component

### PR DESCRIPTION
**Pull request by jasmeets-6575**

### CODE REVIEW

File: src/basic-components/Tooltip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	color?: string;
+	overlayClassName?: string;
+	onOpenChange?: (e: any) => void;
+}
+
+export default function Popover({ children, key, arrow, open, placement, destroyTooltipOnHide, title, className, color, overlayClassName, onOpenChange }: Props) {
+	return (
+		<AntdTooltip
+			key={key}
+			arrow={arrow}
```
Comment (KanishkaRajputd) [inline src/basic-components/Tooltip/index.tsx:27]: arrow should be mandatory

File: src/basic-components/Popover/index.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	return (
+		<AntdPopover
+			open={open}
+			defaultOpen={defaultOpen}
+			content={content}
+			title={title}
+			className={className}
+			zIndex={zIndex}
+			placement={placement}
+			arrow={arrow}
```
Comment (KanishkaRajputd) [inline src/basic-components/Popover/index.tsx:48]: arrow should be initially true

**Review by KanishkaRajputd — CHANGES_REQUESTED** [review]
please resolve changes

**Review by jasmeets-6575 — COMMENTED** [review]
Done

**Review by KanishkaRajputd — APPROVED** [review]

**Review by KanishkaRajputd — APPROVED** [review]

**Pull request merged**